integration/service/test/test_cron_use_case_plot.py

```diff
@@ -0,0 +1,67 @@
+import yaml
+import sys
+from glob import glob
+from pandas import DataFrame, to_datetime
+
+report_names = glob('geopm-report-*.yaml')
+report_list = []
+for rn in report_names:
+    with open(rn) as fid:
+        report = yaml.load(fid, Loader=yaml.SafeLoader)
+    if report is None:
+        sys.stderr.write(f'Warning: {rn} could not be parsed\n')
+        continue
+    df_row_keys = ['host', 'sample-time-first', 'sample-time-total',
+                   'sample-count', 'sample-period-mean', 'sample-period-std']
```

Comment:
Alternatively, if you think the non-`metrics` fields may ever change in the future, it could be something like `df_row_keys = set(report) - {'metrics'}`